src/rustdoc/membership.md

```diff
@@ -0,0 +1,121 @@
+  tasks, at least within the scope of their "rustdoc area". They should plausibly be able to mentor
+  others on simple PRs.
+- "Cordiality" -- rustdoc team members will be part of the Rust organization and are held to a
+  higher standard with respect to the [Code of Conduct][CoC]. They should not only obey the
+  letter of the CoC but also its spirit.
+
+[CoC]: https://www.rust-lang.org/policies/code-of-conduct
+
+Being promoted to member implies a number of privileges:
+
+- Members have `r+` (approve a pull request) privileges and can do reviews (they are expected to
+  use those powers appropriately, as discussed previously). They also have access to control
+  perf/rustc-timer and other similar bots. See the documentation for `bors` and `r+`
+  [here](https://rustc-dev-guide.rust-lang.org/contributing.html#r-1).
+
+  Tip: some baseline rules around bors permissions are: don't do a `try` build unless you have
+  done a check for malicious code first and don't `r+` unless you are reasonably confident that
+  you can effectively review the code in question.
+- Rustdoc team members are members of the Rust organization so they can modify labels and be
+  assigned to issues.
+- Members become a part of the `rust-lang/rustdoc` team on GitHub, so that they receive pings
+  when people are looking to address the team as a whole.
+- Members are [listed] on the rust-lang.org web page.
+
+It also implies some obligations (in some cases, optional obligations):
+
+- Members may take part in various other [maintainer activities] to help the team.
+- Members are held to a higher standard than ordinary folk when it comes to the [Code of
+  Conduct][CoC].
+
+[listed]: https://www.rust-lang.org/governance/teams/dev-tools#team-rustdoc
+
+## What it means to be a rustdoc member
+Once you're a member of the rustdoc team, a number of events will happen:
+
+- You will gain access to a private Zulip stream, where internal discussions happen.
+- You will also be subscribed to the `all@rust-lang.org` mailing list. See
+  [this file](https://github.com/rust-lang/team/blob/HEAD/teams/all.toml) to check how subscriptions
+  to mailing lists work. It's a very low-volume mailing list (maybe a few emails per year), it's a
+  way to communicate things to all contributors. We will not send you spam from this address.
+
+## Roles in the rustdoc team
+
+Rustdoc has multiple different areas and team members are not working on all of them. Currently
+we have three main areas:
+
+- Front-end: Everything related to HTML/CSS/JS
+- JSON backend: Work on the `--output-format=json` backend.
+- Internals: The internals of rustdoc: interacting with the compiler, doctests, generating
+  rustdoc internal code representation, parsing command line arguments, lints, etc.
+
+For now, only the front-end has an official status in the
+[team repository](https://github.com/rust-lang/team). If you're interested to be part of this
+group, you can ask to be added into it.
```

Comment:
I'm sorry if I sound a bit pedantic here but the phrasing is still elusive to me.

> If you're interested to be part of this
group, **you** [emphasis: mine] can ask to be added into it.

"You" as in ~anybody or as in T-rustdoc member? If it's truly ~anybody then this would mean we won't have any proper procedures/rules for T-rustdoc-frontend.

T-rustdoc-frontend also grants r+ & perf rights + access to dev desktops and so it should have similar requirements to T-rustdoc. If that's what you meant anyway, please clarify :)

Reply:
Thanks, now I understand better your expectations. Let's see if I can live up to them! =D